### PR TITLE
Daaaaark mode in typography

### PR DESCRIPTION
**Pull request by jaketrent**
:dracula:

### CODE REVIEW

File: docs/src/components/box.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import Theme, { useTheme } from '@pluralsight/ps-design-system-theme'
+import cx from 'classnames'
+import * as React from 'react'
```
Comment (danethurber) [inline docs/src/components/box.tsx:3]:
```suggestion
import React from 'react'
```

🤷‍♂️

File: docs/src/components/typography/monospace-font.module.css [hunk 2]
```diff
@@ -13,13 +16,14 @@
   text-align: center;
 }
 .fontLetter {
+  composes: glyphs;
```
Comment (danethurber) [inline docs/src/components/typography/monospace-font.module.css:19]: 🍾

**Review by danethurber — APPROVED** [review]

File: docs/src/components/box.tsx (hunk 1 above)
Comment (jaketrent) [reply docs/src/components/box.tsx:3]: Apparently the `* as` is going to be the only supported import style in a later react version.

https://twitter.com/sebmarkbage/status/1250285076014747648

File: docs/src/components/typography/monospace-font.module.css (hunk 2 above)
Comment (jaketrent) [reply docs/src/components/typography/monospace-font.module.css:19]: Just thought I'd introduce another class composition pattern into this codebase. :)

Lol, I'm not sure what your emoji means.

**Pull request merged**